### ISSUE TITLE
does MVCGrid.Net support ASP NET Core ?

### ISSUE THREAD

**Issue by duynguyen0511** [joeharrison714/MVCGrid.Net#82]
Can you confirm if MVCGrid.Net support ASP NET Core ? If not, when will it support ?

**Comment by joeharrison714** [comment]
No, MVCGrid.Net would not work on .net core. I haven't yet looked into this or determined the best approach for making it work. Any thoughts people have would be appreciated.
